src/algorand/AlgorandTools.ts

```diff
@@ -156,3 +157,14 @@ export async function makeCurrencyTools(
 }
 
 export { makeCurrencyEngine } from './AlgorandEngine'
+
+export async function updateInfoPayload(
```

Comment:
Can we keep the `export { makeCurrencyEngine } from './XyzEngine'` lines at the very bottom of each file?